src/components/SavedNewsHeader/SavedNewsHeader.jsx

```diff
@@ -0,0 +1,28 @@
+function SavedNewsHeader({ savedArticles, keywords = [] }) {
+  const formatKeywords = (keywords) => {
+    const length = keywords.length;
+    if (length === 1) return keywords[0];
+    if (length === 2) return `${keywords[0]}, ${keywords[1]}`;
+    if (length === 3) return `${keywords[0]}, ${keywords[1]}, ${keywords[2]}`;
+    if (length > 2)
+      return `${keywords[0]}, ${keywords[1]}, and ${length - 2} other${
+        length - 2 > 1 ? "s" : ""
+      }`;
+    return "";
+  };
+
+  return (
+    <div className="saved__news-header">
+      <p className="saved__news--page">Saved articles</p>
```

Comment:
**Modifier** should be used only with its modifiable **basic** class: `block__element block__element_modifier-key_modifier-value`. Elements should get only **modifying styles** from modifier, and all other styles should be got from the basic class.


**Please, correct it everywhere in the code**